src/components/AddFolderButton/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+
+import { Button } from 'antd';
+import { FolderAddFilled } from '@ant-design/icons';
+
+import AddFolderModal from 'components/AddFolderModal';
+
+import './styles.scss';
+
+function AddFolderButton() {
+  const [isAddFolderModalVisible, setIsAddFolderModalVisible] = useState(false);
+
+  return (
+    <>
+      <AddFolderModal
+        handleCancel={() => setIsAddFolderModalVisible(false)}
```

Comment:
lets avoid inlining functions. there should be a `handleCloseModal`  fn that does this.